src/utils/firestore/index.ts

```diff
@@ -1,7 +1,7 @@
 import firebase from 'firebase/app'
 import 'firebase/firestore'
 
-const firebaseConfig = {
+export const firebaseConfig = {
```

Comment:
This shouldn't need to be exported... It's intentionally only exposing `firestore` - you shouldn't be using the config anywhere else.

Reply:
artefacts from when I tried to work with `reactfire`, the config shouldn't be committed in the first place if you want to actually discuss this

Reply:
In Firebase documentation the frontend config can be exposed, if you actually want to discuss this. https://firebase.google.com/docs/web/setup

Reply:
>Note: The Firebase config object contains unique, but non-secret identifiers for your Firebase project.
Visit Understand Firebase Projects to learn more about this config object.

I stand corrected!

Reply:
Just remove the export, and this would be A-OK for me

```suggestion
const firebaseConfig = {
```